prismicio/slice-machine
# Simulate repeatable links

**Resolves**: 
https://linear.app/prismic/issue/DT-2428/aauser-i-can-use-the-slice-simulator-with-the-new-repeatable-links

### Description





### Checklist





### Preview



### How to QA [^1]







[^1]:
	Please use these labels when submitting a review:
	:question: #ask:&ensp;Ask a question.
	:bulb: #idea:&ensp;Suggest an idea.
	:warning: #issue:&ensp;Strongly suggest a change.
	:tada: #nice:&ensp;Share a compliment.

Add link and repeatable links and see how the mocks for the slice change and have the proper values.
